proto/cronos/query.proto

```diff
@@ -24,6 +25,11 @@ service Query {
     // ReplayBlock replay the eth messages in the block to recover the results of false-failed txs.
     rpc ReplayBlock(ReplayBlockRequest) returns (ReplayBlockResponse) { }
 
+    // Params queries all parameters.
+    rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
+        option (google.api.http).get = "/cronos/v1/params";
```

Comment:
not sure if we'd enable `module_query_safe` here

Reply:
what's this option used for? I didn't see such options in the context.

Reply:
also new, but you could trace [here](https://github.com/cosmos/cosmos-sdk/blob/main/proto/cosmos/query/v1/query.proto)

Reply:
seems indeed safe for module queries by itself.
but we may upgrade this in another pr.